packages/runtime/runtime-definitions/src/id-compressor/identifiers.ts

```diff
@@ -3,48 +3,6 @@
  * Licensed under the MIT License.
  */
 
-/**
- * Type-safe identifiers for specific use cases.
- */
-
-/**
- * A 128-bit Universally Unique IDentifier. Represented here
- * with a string of the form xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx,
- * where x is a lowercase hex digit.
- * @alpha
- */
-export type UuidString = string & { readonly UuidString: "9d40d0ae-90d9-44b1-9482-9f55d59d5465" };
-
-/**
- * A version 4, variant 2 uuid (https://datatracker.ietf.org/doc/html/rfc4122).
- * @internal
- */
-export type StableId = UuidString & { readonly StableId: "53172b0d-a3d5-41ea-bd75-b43839c97f5a" };
-
-/**
- * A StableId which is suitable for use as a session identifier
- * @internal
- */
-export type SessionId = StableId & { readonly SessionId: "4498f850-e14e-4be9-8db0-89ec00997e58" };
-
-/**
- * Edit identifier
- * @alpha
- */
-export type EditId = UuidString & { readonly EditId: "56897beb-53e4-4e66-85da-4bf5cd5d0d49" };
-
-/**
- * Scoped to a single edit: identifies a sequences of nodes that can be moved into a trait.
- *
- * Within a given Edit, any DetachedSequenceId must be a source at most once, and a destination at most once.
- * If used as a source, it must be after it is used as a destination.
- * If this is violated, the Edit is considered malformed.
- * @alpha
- */
-export type DetachedSequenceId = number & {
-	readonly DetachedSequenceId: "f7d7903a-194e-45e7-8e82-c9ef4333577d";
-};
-
 /**
```

Comment:
Do we still use InternedStringId?

Reply:
Nope, good catch. Removed in latest commit.